session/bootstrap.go

```diff
@@ -2062,10 +2061,9 @@ func doDMLWorks(s Session) {
 			vVal = variable.AssertionFastStr
 		case variable.TiDBEnableMutationChecker:
 			vVal = variable.On
-		case variable.TiDBEnablePaging:
-			vVal = variable.BoolToOnOff(variable.DefTiDBEnablePaging)
```

Comment:
This is an unnecessary branch, because the default is DefTiDBEnablePaging anyway.